charts/corda/templates/flow-worker.yaml

```
@@ -42,6 +42,8 @@ spec:
         {{- include "corda.workerEnv" . | nindent 10 }}
         args:
         {{- include "corda.workerKafkaArgs" . | nindent 10 }}
+          - -spassphrase=$(PASSPHRASE)
+          - -ssalt=$(SALT)

```

@davidcurrie 
FYI, this PR change the behaviour in that it throws an exception if you haven't configured a "Secrets Service" instead of a warning (it's mandatory for all workers as otherwise they would not be able to resolve sensitive configuration).

---

You'll need to replicate the following for those environment variables to actually be available though:
https://github.com/corda/corda-runtime-os/blob/release/os/5.0/charts/corda/templates/crypto-worker.yaml#L44

If you're now making them available on all workers, it's probably better just to add `dbSaltAndPassphraseEnv` to `workerEnv`. Might also be worth expanding `workerKafkaArgs` to `workerArgs` and adding the salt and passphrase in there.

---

(As an aside: the configuration for the workers has diverged much less than I thought it might. I'm tempted to actually create a single parameterised template for a worker but that's a bigger piece of work.)

---

So I'm guessing this works because we're effectively passing empty values for this? 
`dbSaltAndPassphraseEnv` doesn't seem right as this has nothing to do with the DB.